services/storage/decorators/authn.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
```

Comment:
The storage layer, however, only tries to load the User from the database if a handler _specifically requests it_ using one of these decorators.